### PR TITLE
CONTRIBUTING.md: instructions for making a release

### PR DESCRIPTION
**Pull request by jba**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,3 +34,36 @@ Guidelines](https://opensource.google/conduct/).
...
+cp devtools/pre-push-hook.sh .git/hooks/pre-push
+```
+
+### Creating a new release
+
+This repo consists of a single Go module.
+To increase the minor or patch version of the module:
+
+1. Determine the desired tag, using `git tag -l` to see existing tags
+   and incrementing as appropriate. We will call the result TAG in
```
Comment (eliben) [inline CONTRIBUTING.md:53]: Mention that TAG naming should be `vX.Y.Z`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,3 +34,36 @@ Guidelines](https://opensource.google/conduct/).
...
+   pass the `--no-verify` flag to `git push`.
+4. Submit the PR when approved. _No other PRs should be submitted until
+   the following steps have been completed._
+5. Run `git pull` to get the submitted PR locally. You should be on main.
+6. Run `git tag TAG` to tag the repo locally.
+7. Run `git push origin TAG`. If the pre-push hook complains here, something
+   is wrong; stop and review.
+8. Review the changes since the last release to prepare release notes.
+   If the last release tag was PREVTAG, then run `git log PREVTAG..`.
+9. Use the [GitHub UI](https://github.com/google/generative-ai-go/releases) to
```
Comment (eliben) [inline CONTRIBUTING.md:66]: The release name should just be TAG -- we had some inconsistency here before, so it's good to mention

**Review by eliben — APPROVED** [review]
Great, thanks for putting this in writing

File: CONTRIBUTING.md (hunk 1 above)
Comment (jba) [reply CONTRIBUTING.md:53]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jba) [reply CONTRIBUTING.md:66]: Done.

**Pull request merged**